command/ssh.go

```diff
@@ -45,6 +47,15 @@ func (c *SSHCommand) Run(args []string) int {
 	if err := flags.Parse(args); err != nil {
 		return 1
 	}
+
+	if os.Getenv("VAULT_STRICT_HOST_KEY_CHECKING") != "" {
+		strictHostKeyChecking = os.Getenv("VAULT_STRICT_HOST_KEY_CHECKING")
```

Comment:
I think the vars should be `VAULT_SSH_......` as they're specific to this command. I realize that they're rather long, but I don't think they'll be toggled often.